eli5/sklearn/score_decrease.py

```diff
@@ -0,0 +1,230 @@
+"""
+A sklearn-compatible object for computing score decrease-based
+feature importances.
+"""
+from functools import partial
+from typing import List
+
+import numpy as np  # type: ignore
+
+from sklearn.model_selection import check_cv  # type: ignore
+from sklearn.utils.metaestimators import if_delegate_has_method  # type: ignore
+from sklearn.utils import check_array, check_random_state  # type: ignore
+from sklearn.base import (  # type: ignore
+    BaseEstimator,
+    MetaEstimatorMixin,
+    clone,
+    is_classifier
+)
+from sklearn.metrics.scorer import check_scoring  # type: ignore
+
+from eli5 import score_decrease
+
+
+class ScoreDecreaseFeatureImportances(BaseEstimator, MetaEstimatorMixin):
+    """Meta-transformer which exposes feature_importances_ attribute
+    based on average score decrease.
+
+    ScoreDecreaseFeatureImportances instance can be used instead of
+    its wrapped estimator, as it exposes all estimator's common methods like
+    ``predict``.
+
+    There are 3 main modes of operation:
+
+    1. prefit=True (pre-fit estimator is passed). You can call
```

Comment:
In `CalibratedClassifierCV` we went with `cv='prefit'` to express these as mutually exclusive options.